python/cudf/cudf/_lib/orc.pyx

```diff
@@ -313,32 +162,38 @@ cdef compression_type _get_comp_type(object compression):
         raise ValueError(f"Unsupported `compression` type {compression}")
 
 cdef tuple _get_index_from_metadata(
-        map[string, string] user_data,
+        vector[map[string, string]] user_data,
         object names,
         object skip_rows,
         object num_rows):
-    json_str = user_data[b'pandas'].decode('utf-8')
+
+    # TODO: consider metadata from more than the first file?
```

Comment:
Should we convert this TODO and make this into a documented behavior/limitation?

Reply:
Yeah I'll put it into the docstring.

Reply:
Done.